### PR TITLE
Improve Capistrano configuration

### PR DESCRIPTION
**Pull request by skddc**
See commit messages and descriptions. Thanks.

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 set :repo_url, ENV['CAPISTRANO_DEPLOY_REPO_URL'] || 'https://github.com/cantino/huginn.git'
 
 # Default branch is :master
-set :branch, ENV['BRANCH'] || 'master'
+set :branch, ENV['CAPISTRANO_DEPLOY_BRANCH'] || 'master'
```
Comment (cantino) [inline config/deploy.rb]: Will this rename break people's existing deployment setup?

Comment (dsander) [inline config/deploy.rb]: Having the same naming scheme for the variables makes sense, but I think @cantino is right. We should keep `BRANCH` as the second fallback here.

@skddc Can you also change the variable in the [documentation](https://github.com/huginn/huginn/blob/master/doc/manual/capistrano.md#3-run-capistrano)?

Comment (skddc) [inline config/deploy.rb]: Good point, I added a fallback to the old name.

**Comment by cantino** [comment]
Looks great, thanks @skddc!

**Pull request merged**